### ISSUE TITLE
Bug: Windows: deno stays idle for some seconds after using puppeteer

### ISSUE THREAD

**Issue by c-antin** [denoland/deno#20360]
When testing puppeteer with deno on Windows 10, I noticed that the deno.exe stays idle for some seconds.

Code to reproduce:
```ts
import puppeteer from "npm:puppeteer-core@21.1.1";

//deno run --allow-read --allow-env --allow-write=./temp,./export --allow-run --allow-net=127.0.0.1 --node-modules-dir=false main.ts

console.time("pdf");
// deno-lint-ignore no-explicit-any
(puppeteer as any).configuration = { temporaryDirectory: "./temp" };

const browser = await puppeteer.launch({
  executablePath: String
    .raw`C:\Program Files\Google\Chrome\Application\chrome.exe`,
  handleSIGHUP: false,
});
const page = await browser.newPage();

await page.goto("https://news.ycombinator.com", {
  waitUntil: "networkidle2",
});
// page.pdf() is currently supported only in headless mode.
// @see https://bugs.chromium.org/p/chromium/issues/detail?id=753118
const buffer = await page.pdf({
  format: "letter",
});
await Deno.writeFile("./export/hn.pdf", buffer);

await browser.close();
console.timeEnd("pdf");
// Deno.exit(0); //required because deno stays idle for some seconds

```

Expected behavior: the process should exit immediately after awaiting `browser.close();`.

A similar node script behaves as expected:
```js
import puppeteer from 'puppeteer-core';

//node main.mjs

(async () => {
    console.time("pdf");

    puppeteer.configuration = { temporaryDirectory: "./temp" };

    const browser = await puppeteer.launch({
        executablePath: String
            .raw`C:\Program Files\Google\Chrome\Application\chrome.exe`,
        handleSIGHUP: false,
    });
    const page = await browser.newPage();

    await page.goto("https://news.ycombinator.com", {
        waitUntil: "networkidle2",
    });
    // page.pdf() is currently supported only in headless mode.
    // @see https://bugs.chromium.org/p/chromium/issues/detail?id=753118
    await page.pdf({
        path: "./export/hn_node.pdf",
        format: "letter",
    });

    await browser.close();
    console.timeEnd("pdf");
})();

```

**Comment by guillaume86** [comment]
Yes there's a 30 sec timeout, same issue as https://github.com/denoland/deno/issues/20179

**Comment by kt3k** [comment]
This should be fixed by #25387 

The fix is available in 2.0.0-rc.2 or above. You can upgrade to RC by the command:

```
deno upgrade
deno upgrade rc
```
